openapi.yaml

```diff
@@ -0,0 +1,149 @@
+openapi: '3.0.3'
+info:
+  title: Genius Yield Bot API
+  description: Open API Specificaion of the Genius Yield Bot API for trading strategy integration.
+  version: '0.1.0'
+components:
+  securitySchemes:
```

Comment:
To secure the communication between the MMB and the Active Strategy, which is using the Bot API served by the MMB - we must introduce some level of security features. At the very minimum we need some API Keys to make sure that no unauthorized third party could place or cancel orders by calling the MMB API.

Reply:
Discussed with @MatD9 and @sourabhxyz. This is going to be tackled.